module.modulemap

```diff
@@ -1,4 +1,4 @@
-module libmongoc [system] {
+module mongoc [system] {
```

Comment:
The header included here is the one in this repository, the so-called "bridging header". I left it the same name because it was easy. If I were going to rename it I might choose something like `header_name_that_makes_no_difference.h`